backend/database/databasefile.sql

```diff
@@ -53,12 +53,29 @@ CREATE TABLE IF NOT EXISTS `presetsDatabase`.`presets` (
 ENGINE = InnoDB
 DEFAULT CHARACTER SET = utf8;
 
+-- -----------------------------------------------------
+-- Table `presetsDatabase`.`simplepresets`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `presetsDatabase`.`simplepresets` (
+  `ID` INT(11) NOT NULL,
+  `Image` CHAR(50) NULL DEFAULT '',
+  `camera_ID` INT(11) NOT NULL,
+  PRIMARY KEY (`ID`, `camera_ID`),
```

Comment:
The primary key is only for the ID, not with the camera_ID. This was already wrong in the presets table, but only now i notice it.